.github/semantic.yml

```diff
@@ -0,0 +1,8 @@
+# This is the configuration file for the Semantic Pull Requests GitHub App.
+# Reference: https://github.com/zeke/semantic-pull-requests
+---
+enabled: true
+titleAndCommits: true
+anyCommit: true
+allowMergeCommits: true
+allowRevertCommits: true
```

Comment:
Setting `titleAndCommits` to `true` makes this unnecessarily strict. Since our default strategy is to squash, the PR title is usually the only thing that matters, so why enforce this on commits as well?

I think `titleOnly` would be fine. But perhaps even better is to just leave the default behavior in place:

> By default, only the PR title OR at least one commit message needs to have semantic prefix.

Reply:
Yeah...this file was only added after the PR posted...as a means to see if it was required in order to get the App to function. It was later discovered that the service behind the GitHub App is down and this config file has nothing to do with it. The original intent was to keep the default behavior...so this file will be removed. It's still not clear if switching, even temporarily, to an alternative solution is the path to go here...vs. simply waiting for the issue to resolve on it's own (there is recent activity on [the issue](https://github.com/zeke/semantic-pull-requests/issues/183)